kyo-data/shared/src/main/scala/kyo/KyoException.scala

```diff
@@ -2,17 +2,14 @@ package kyo
 
 import kyo.*
 import kyo.Ansi.*
+import kyo.internal.Environment
 import scala.util.control.NoStackTrace
 
-class KyoException private[kyo] (
-    message: Text | Null = null,
-    cause: Text | Throwable | Null = null
-)(using val frame: Frame) extends Exception(
-        Option(message) match
-            case Some(message) => message.toString;
-            case _             => null,
+class KyoException private[kyo] (message: Text = "", cause: Text | Throwable = "")(using val frame: Frame)
+    extends Exception(
+        message.toString,
         cause match
-            case cause: Throwable => cause;
```

Comment:
:scream_cat: is the `;` the cause of the trouble? I'm so sorry :crying_cat_face:

Reply:
No, it's harmless :)